### PR TITLE
Ingestion: NEJLT 2023

### PR DESCRIPTION
**Pull request by anthology-assist**
1. [x] In the Github sidebar, add the PR to work items and the current milestone
1. [x] In the Github sidebar, under "Development", make sure to link to the corresponding issue
1. [x] Make sure the branch is merged with the latest `master` branch
1. [x] Ensure that there are editors listed in the `<meta>` block
1. [ ] For workshops, add a `<venue>ws</venue>` tag to its meta block
1. [ ] For workshops, add a backlink from the main event's `<event>` block
1. [ ] Add events to their relevant SIGs
1. [x] Look at the venue listing for prior years, and ensure that the new volume titles are consistent. You can do this by clicking on the venue name from a paper page, which will take you to the vendor listing.
1. [x] Navigate to the event page preview (e.g., https://preview.aclanthology.org/icnlsp-ingestion/events/icnlsp-2021/), and page through, to see if there are any glaring mistakes
1. [x] Skim through the complete listing, looking for mis-parsed author names.
1. [ ] Download the frontmatter and verify that the table of contents matches at least three randomly-selected papers
1. [x] Download 3–5 PDFs (including the first and last one) and make sure they are correct (title, authors, page numbers).

### CODE REVIEW

**Comment by mbollmann** [comment]
LGTM, except that the full volume PDF doesn’t seem to be included? (The `data/nejlt-09/proceedings/cdrom/nejlt-2023.1.pdf`)

**Comment by mjpost** [comment]
@anthology-assist can you comment here? What happened to the full volume PDF?

File: data/xml/2023.nejlt.xml [hunk 1]
```diff
@@ -0,0 +1,244 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2023.nejlt">
+  <volume id="1" ingest-date="2024-08-06" type="proceedings">
+    <meta>
+      <booktitle>Northern European Journal of Language Technology, Volume 9</booktitle>
+      <editor><first>Leon</first><last>Derczynski</last></editor>
+      <publisher>Northern European Association of Language Technology</publisher>
+      <address>Copenhagen, Denmark</address>
```
Comment (mbollmann) [inline data/xml/2023.nejlt.xml:8]: I realized that this is a mistake that also needs to be corrected in previous issues; it should be:

```xml
<publisher>Linköping University Electronic Press</publisher>
<address>Linköping, Sweden</address>
```

(I could open a PR for previous issues.)

**Comment by mbollmann** [comment]
I tried to run the ingestion myself locally, but I feel I’m missing something or don’t know which script to use, since I’m not getting the same XML file as here with `bin/ingest.py`. :)

Would be nice if full volume PDF could be added as well. I’m happy for pointers if there was something wrong in the ingestion material regarding that.

**Comment by mjpost** [comment]
Ack, I'm really sorry ingestion here came to a complete standstill.

File: data/xml/2023.nejlt.xml (hunk 1 above)
Comment (mjpost) [reply data/xml/2023.nejlt.xml:8]: Do you want to just fix these and add them to this PR?

**Comment by mjpost** [comment]
@mbollmann can you post the full-volume PDF somewhere (to this post, if possible), and I will ingest this?

**Comment by mjpost** [comment]
Here is the code that looks for book names: https://github.com/acl-org/acl-anthology/blob/586863a257a565e5047f7b15219204e4529ad50e/bin/ingest.py#L291

It seems we need to add a new pattern. I just did that. We can fix this manually this time if I get the PDF.

**Comment by mbollmann** [comment]
Full issue PDF: https://nejlt.ep.liu.se/issue/view/407/337

I can also update the name in the script that produces the materials of course, I have just been re-using what Leon used in previous years to produce these materials. :)

**Review by mjpost — APPROVED** [review]

**Comment by mbollmann** [comment]
LGTM now, thank you!

**Pull request merged**